app/views/experiences/new.html.erb

```diff
@@ -18,24 +18,7 @@
             </tr>
             <tr>
                 <td class="experience-edit-label">Image:</td>
-                <!--<td>< %= file_field_tag 'images[]', :multiple => true, :accept => 'image/jpeg,image/png, image/jpg, image/gif', class: "form-control" %>-->
```

Comment:
I think this is a recent change made by @kyfr7525
, so you would have to include this change. You probably did not pull the latest changes from the master branch